src/admin_tools/main.py

```diff
@@ -131,15 +130,14 @@ def _reindex_ws_type(args):
     # - Iterate over all workspaces
     #   - For each workspace, list objects
     #   - For each obj matching args.type, produce a reindex event
-    ws_client = WorkspaceClient(url=config()['kbase_endpoint'], token=config()['ws_token'])
     evtype = 'INDEX_NONEXISTENT'
     if args.overwrite:
         evtype = 'REINDEX'
     objids = []
     for wsid in range(args.start, args.stop + 1):
         wsid = int(wsid)
         try:
-            infos = ws_client.generate_obj_infos(wsid, admin=True)
+            infos = config()['ws_client'].generate_obj_infos(wsid, admin=True)
```

Comment:
Caching the ws client in the config instead of having to reinitialize it everywhere